packages/core/solidity/src/zip-hardhat-viem.ts

```
@@ -0,0 +1,218 @@
+    const lines = ['const connection = await hre.network.create();', 'const { viem } = connection;'];
+    if (c.upgradeable) {
+      lines.push('const upgradesApi = await upgrades(hre, connection);');
+    }
+    return lines;
+  }
+
+  private getTestCase(c: Contract, opts?: GenericOptions): Lines[] {
+    const argNames = c.constructorArgs.map(a => a.name);
+    return [
+      `test("${c.name}", async t => {`,
+      spaceBetween(this.declareVariables(c.constructorArgs), this.getDeployLines(c, argNames), this.getAssertions(c, opts)),
```

ESLint

[error] Replace `this.declareVariables(c.constructorArgs),·this.getDeployLines(c,·argNames),·this.getAssertions(c,·opts)` with `⏎········this.declareVariables(c.constructorArgs),⏎········this.getDeployLines(c,·argNames),⏎········this.getAssertions(c,·opts),⏎······`

(prettier/prettier)